larpandoracontent/LArControlFlow/MasterAlgorithm.cc

```diff
@@ -418,6 +418,29 @@ StatusCode MasterAlgorithm::RunCosmicRayHitRemoval(const PfoList &ambiguousPfos)
     {
         const ClusterList clusterList(pPfoToDelete->GetClusterList());
         const VertexList vertexList(pPfoToDelete->GetVertexList());
+
```

Comment:
I think this entire function is just about under the line limit for a single function. If the x0 reset gets any bigger, it could be extracted as a separate function, but I think it's OK right now.